doc/dev_guide/internals/msgpack_extensions.rst

```diff
@@ -233,3 +240,141 @@ the server response will look like this:
             00                              MP_UINT = error number
             05                              MP_ERROR_ERRCODE
             0a                              MP_UINT = eror code ER_SPACE_EXISTS
+
+
+..  _msgpack_ext-datetime:
+
+**********************************
+The DATETIME type
+**********************************
+
+Since version :doc:`2.10.0 </release/2.10.0>`.
+The MessagePack EXT type ``MP_EXT`` together with the extension type
+``MP_DATETIME`` is a header for values of the DATETIME type.
+It creates a container of 8 or 16 bytes long payload.
```

Comment:
"long payload" -- is it a term, or we just mean "8 or 16 bytes long"? If the second case, please rephrase.